### ISSUE TITLE
Fix false positives for quadruple slashes comments in value-keyword-case

### ISSUE THREAD

**Issue by AlexWayfer** [stylelint/stylelint#4610]
> Clearly describe the bug

There are false positives for "No" in single-line comment with 4 `/` inside Set variable… :smiley: 

> Which rule, if any, is the bug related to?

`value-keyword-case`

> What CSS is needed to reproduce the bug?

```scss
$breakpoints: (
  // Extra small devices (portrait phones, less than 576px)
  //// No media query since this is the default
  // Small devices (landscape phones, 576px and up)
  sm: 576px,
  // Medium devices (tablets, 768px and up)
  md: 768px,
  // Large devices (desktops, 992px and up)
  lg: 992px,
  // Extra large devices (large desktops, 1200px and up)
  xl: 1200px
);
```

> What stylelint configuration is needed to reproduce the bug?

e.g.

```json
{
  "rules": {
    "value-keyword-case": "lower"
  }
}
```

> Which version of stylelint are you using?

`13.2.0`

> How are you running stylelint: CLI, PostCSS plugin, Node.js API?

CLI (reproducible in online demo)

> Does the bug relate to non-standard syntax (e.g. SCSS, Less etc.)?

Yes, it's related to SCSS variables and sinle-line comments.

> What did you expect to happen?

No warnings to be flagged.

> What actually happened (e.g. what warnings or errors did you get)?

The following warnings were flagged:

```
Expected "No" to be "no" (value-keyword-case)
```

**Comment by jeddy3** [comment]
@AlexWayfer Thanks for the report and for using the template.

I can reproduce this [in the demo](https://stylelint.io/demo#N4Igxg9gJgpiBcIAkBbAhgB3gAgBQB0A7bbAenNOwHUIAnKIgSgG4iQAaEAZwE9CAXNAA8E3MFy4dwEQgDMAlgHNRwIiXwhaAVwA2MLhpyriJbBrT8AtNr2XCES1sIBrewHdCh7P20x2a0w0AIx0IMGc7BxgUDH4eLx8tPwD1aVDaSMt5QgA3NB15KEsACxghBN9-E1TIFBQYAUzo2PiEb0qUsxBYMB00Wgt5GUsQsIj7SygtDAKwCxhLDFoIDBhafnl9LwBtTpJE5OrTY1NTrqV7WhgdvbONSEIuGDAtDZyFqZn5Of59Szd5PxipN5LJZGsGlY8jokgYQLcSABdW4AX06iKqpw0PT6Aw2w1G4UyXGKdCBaEIRSWKzWcUsEHetFohS2bQOmMCIFkMissjQKHkOh4mU+s3mdn5rJw7M6Gm5jT5AqFmQFEmyiksigaa2+lkVguFzhgPDcdAYbI6RzlTjA+MIljmOjAmWy0MKFSSHNSshtdod+WdEycXAwaDAMCKNLxdA9hyxXN9Q3tBUIMH6moGUE2jQm9kegkp-SKWautqTsa9XSNPFkA3qk2euMGwwm8hiZIp-ArspA9SzaD1af4WiuEvrQZc7k8Fs9PYmsC44cp6ssIeeoO+gNa0st8fn0zFv0sFiy7fW1l0Uvas6tIH3Xx+CyeeltdDhO5ve6iMTpXAgI-DbtbwmMofAHJ4BUgUJHiAr8slyfJCkmf8QifPoSQdCA6khd9rzjTlqVWdZhQnVwIA8WDOWfZ5+DoRYni0KAHF6NAJEyJwyIomd8NSajXwyDAGKYywYD0eocwcDip0o3jRJoui4lWdjJ3I6cPx4rouB8FdcxgNwU2ubjKw0JxAWUzi1Lw4yQBPGwFmaOkDJGGBuSuG4jlSfI3DQHg4UrEgTjOVIynDWJ3KC+NCWcPQ2LQWRfgyLhJTHBYopivyEU5BRaC0ux9F+c1MoxTKNAuOhDJwXYPLuGz4rWTDsIEDQitRdFrNsy8Uv9J4vA0UI3DWDR2qsOyupDMMFjihLepsnRvN81d1VsAyhp7DrbAg74IGgvL9OySa6toGavJ8jLbyih1Qi4FcggGcMRJ-YUnKCFzypm1NGVW87QiJXoIGuwgNVuibdqcqbBracw5tOr7Ip+iI-oBoG7oWVM9tTZzXIqroToWlBdA2SwVo4HsLsRm6UdXUN7perHjuhhakeW-bYc5C6aUICmQbRsHDvp+auEsfGdEJ4nrPZ1ZOcBkZKfG+7waOyHZoFxbAeZ1NWdSCWGi5+65dS163KV3GzvjKC6NKIRuuxvryIh6zzYyS2iYaRQgRmkkyU1rpanEqwHKe-bMbetoqqCqGBe9gKSpAEKYDC0PMu9eQcqsVMtIjZrqqRfzOTKo3KqTzS4j0FMrF9ikoDhFrqrRI5itvX3IX+YpAX0amFmya7YH5mGScbrQtKwxZliIukA6JoPaZDwvqojvvc8C8PY6EUKu0T7POQVh1B9olAR6jOIo5q7LcvTgqs4i7AG6v0rFEubGw6vnHDoav3j-jLuWVVxR1dS+HL4RWRLXNqPYcT9GbPaIIFINT62PHzJWH17ZgMbBAv00DpZwOngXHGDNTacnAXiJMIx4armiFtHaPMg4K17njAm8hJ4a37vGQhkCSFjDIZBbaww4E0ONngn+f9vbYlQUQgkpDNrm3tFgw2Ns7wwE+swghoi2EXSZgsJy6ArasLtLhAAjNZHRxCLpgUFCuSR3DLLz18sI7oKi-RSNBtQhBOBrGC2FqLFmSjUhGOGI43hLjcEq3UYwwyhj7HEP8R3YOOCNBIMVuEliYj7QT2erI8KNUTbHyXjVOOCdZ7PxfglBsSTIEfyyinM++VM7wk3jfCKd8H4ZNvrVYpTcmreIaSAL+sBBEaKngA2pV9gFBTrqcep3pEx+KwugJxGM+GuOVqdIW9DQm2J9IQMs0y6jgWiQsoJyyQlix7BsrZ9oK5UxBtguR8T1lTPtFoh6LRQm4QAAzWVOX6QgyU5g9SVv1ZBt5PnENDFcAQpQniCyoRjHpciTYrJFgw45QL7mLH6JCCFfw4Gwvego+qRyvEfNRW4VuvwAnTX4ZHTppVKSQjKW0AATNZPQgMgSWAAF5rAcBOQEMkuiPNSftfR1k+zyAHOCCwI4FhRJBvstxtjRXiqHFKzCO0ZF00QXihJPZFWDklaOb59ZflyIBdq28uqJXDlHKCjF+gsXRJxZqxRIqIxir1VahYAxAYLCjBYOi5KIaLKydS3srqlX6s9TAn1RE-WJWidc2h+DUi6oAI5JFoIHXKFzoUHQpUGgRHikWEp1WGywaa1iZvLjM3ZsrAluL6WskNqb02VrfjWmm6SnWArgjAbaLc276z5XEhwqokYiUpPSWQq5-y0Hujmodd4tAoBehkPQaAszS05csRNtjCBLpXZkUxKYNRbv+guwitJhTGpmqa2xo1BUYwTRvZe8LC2NsXjHPJ68CnPzlJUtO1TCp1NzqkfOj8i7mFfu09eNdRmgNvHxWiGQLDaSCK8VKd0jT8EFtix4LJcXOp7IhuiKHmRoaPL6pDlz5Z1vkYRhDcl+LHn4Kh9D9IY1UfVTPLotyQ3EYyLUII2RY3UdzYGg5Ni+OMao4J4TnH42dvzVS6y-GGyLgaFATsDUhPfKo7mHhHcF2qYKFm6tcyxOKyUwvIj0m6ImarTs0TMSblatsapgVj0XleHeTZl8VHBIwEYsxdCgtr3-Ltma+MqmAtBcuqxQWNrwV2pww6vDPcu2RaorZgSQkohiWbmFxZt6pN+bojF4Sck-aqpbBAQQdoZpMS0KhNz2XLCKWlaxE1EXbGmXLp1m93WQ3QiSJYaspp6DWwGwNTLqRhv9NM45nN8C80Sfcas5F8Y5uTwW7MgNlnVsNo25yLb9m21OafYs3j1kTvyFyh555Bk3kBDriiEAKIgA).

```scss
$map: (
  //// Word
);
```

Gives:

```
2:8 error Expected "Word*" to be "word*" (value-keyword-case)
```

I've labelled the issue as ready to implement. Please consider [contributing](https://stylelint.io/contributing) if you have time.

There are [steps on how to fix a bug in a rule](https://stylelint.io/developer-guide/rules#fix-a-bug-in-a-rule) in the Developer guide.

The issue might be with the rule itself or with [up stream SCSS syntax](https://stylelint.io/developer-guide/syntaxes).

As a workaround, you can avoid using quadruple slashed comments.

**Comment by jeddy3** [comment]
Closing [as stylistic rules are frozen](https://github.com/stylelint/stylelint/issues/5674). The community is welcome to migrate the rule [to a plugin](https://stylelint.io/developer-guide/plugins) and fix this bug.
